### ISSUE TITLE
Feature: Support shell test contexts (e.g. bats-core)

### ISSUE THREAD

**Issue by taqtiqa-mark** [termapps/enquirer#19]
Thanks for all your effort in making enquirer, and for making it open source.

We've struggled to test a bash script that uses `enquirer`.  
Our test framework is [`bats-core`]().

There may be other issues in that context.
However, we believe an initial stumbling block is that neither of the following work:

```bash
enquirer input --message "Project parent folder?" --default "$(pwd)" <<</tmp
enquirer input --message "Project parent folder?" --default "$(pwd)" < <(echo /tmp)
```

This looks like `enquirer` recognized an input device:

```bash
$ enquirer input --default "$(pwd)" --message "Project parent folder?" <(echo /tmp)
error: Found argument '/dev/fd/63' which wasn't expected, or isn't valid in this context

USAGE:
    enquirer input [OPTIONS] --message <MESSAGE>

For more information try --help
```

Appreciate any hints or tips.

**Comment by pksunkara** [comment]
Are you trying to input stuff into enquirer?

The main use case of enquirer is for terminal based interactions. I don't think we allow reading those interactions from files. This is an underlying issue in https://github.com/mitsuhiko/dialoguer.

**Comment by taqtiqa-mark** [comment]
> Are you trying to input stuff into enquirer?

Correct.  This arose in testing scripts that use enquirer. It does look like enquirer would need to support io via pipes and heredocs
